src/vs/workbench/contrib/terminal/browser/links/terminalLinkManager.ts

```diff
@@ -97,6 +97,20 @@ export class TerminalLinkManager extends DisposableStore {
 		this._registerStandardLinkProviders();
 	}
 
+	async getLinks(type: TerminalLinkProviderType, y: number): Promise<ILink[] | undefined> {
+		let provider: ILinkProvider | undefined = undefined;
+		if (type === TerminalLinkProviderType.Word) {
```

Comment:
todo: maybe keep a reference to these so don't have to find them?

Reply:
A `Map<TerminalLinkProviderType, ILinkProvider>` would make more sense if we need to get by key.